create-publication-from-scopus/src/test/resources/expectedResults/expectedAuthorKeywords.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<authorKeywordsTp xmlns:ns6="http://www.elsevier.com/xml/ani/common" xmlns:ns5="http://www.elsevier.com/xml/ani/ait" xmlns:ns7="http://www.elsevier.com/xml/ani/internal" xmlns:ns2="http://www.w3.org/1998/Math/MathML" xmlns:ns4="http://www.elsevier.com/xml/cto/dtd" xmlns:ns3="http://www.elsevier.com/xml/xocs/dtd">
+  <author-keyword xml:lang="eng">
```

Comment:
It's the xmlns:ns5, ns4, ns7 etc that comes in random order that makes String equals not possible.
Tried to research how to remove these fields as I don't see how they can be valuable, but all results showed how to remove them using annotation, which is not possible for us with our auto-generated classes.